src/dguweb/web/controllers/search_controller.ex

```diff
@@ -5,14 +5,12 @@ defmodule DGUWeb.SearchController do
     render conn, :index
   end
 
-
-  def search(conn, %{}), do: showall(conn)
   def search(conn, %{"q" => ""}), do: showall(conn)
-
   def search(conn, %{"q" => query}) do
     result = URI.encode(query) |> Repo.search
     render conn, :search, query: query, results: result.hits.hits, total: result.hits.total
   end
+  def search(conn, %{}), do: showall(conn)
```

Comment:
It's a fair comment.  I've seen lots that does this style of grouping, because that's what you do in Erlang - but I really don't like it because I think it reduces readability a lot.  I'll be in this file again for next PR so I'll change it there.